source/config/configFlags.py

```diff
@@ -238,20 +238,20 @@ def _displayStringLabels(self):
 		}
 
 
-class AddonsAutomaticUpdate(DisplayStringStrEnum):
+class AddonsUpdateMode(DisplayStringStrEnum):
 	NOTIFY = "notify"
 	UPDATE = "update"
 	DISABLED = "disabled"
 
 	@property
 	def _displayStringLabels(self):
 		return {
-			# Translators: This is a label for the automatic update behaviour for add-ons.
+			# Translators: This is a label for the add-on update behaviour for add-ons.
 			# It will notify the user when updates are available.
 			self.NOTIFY: _("Notify"),
-			# Translators: This is a label for the automatic update behaviour for add-ons.
+			# Translators: This is a label for the add-on update behaviour for add-ons.
 			self.UPDATE: _("Update Automatically"),
-			# Translators: This is a label for the automatic update behaviour for add-ons.
+			# Translators: This is a label for the add-on update behaviour for add-ons.
```

Comment:
I think this is a bit misleading - as it doesn't affect the update behaviour from the update panel in the add-on store